modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/winback/WinbackOfferPage.kt

```diff
@@ -1,346 +1,268 @@
-        Spacer(
-            modifier = Modifier.weight(1f),
+        HeartIcon(
+            painter = painterResource(IR.drawable.ic_winback_offer_heart_1),
+            width = maxWidth * 0.728f,
+            height = maxHeight * 0.654f,
+            offsetX = maxWidth * 0.077f,
+            offsetY = maxHeight * 0.225f,
+            gradientColors = gradientColors,
+            alpha = 1f,
+            rotation = 0f,
         )
-        Image(
-            painter = painterResource(IR.drawable.ic_chevron_small_right),
-            contentDescription = null,
-            colorFilter = ColorFilter.tint(MaterialTheme.theme.colors.primaryIcon02),
+        HeartIcon(
+            painter = painterResource(IR.drawable.ic_winback_offer_heart_2),
+            width = maxWidth * 0.75f,
+            height = maxHeight * 0.7f,
+            offsetX = maxWidth * 0.23f,
+            offsetY = maxHeight * 0.14f,
+            gradientColors = gradientColors,
+            alpha = 0.2f,
+            rotation = 7.84f,
         )
     }
 }
 
 @Composable
-private fun Divider(
-    modifier: Modifier = Modifier,
+private fun HeartIcon(
+    painter: Painter,
+    width: Dp,
+    height: Dp,
+    offsetX: Dp,
+    offsetY: Dp,
+    gradientColors: Pair<Color, Color>,
+    alpha: Float,
+    rotation: Float,
 ) {
-    MaterialDivider(
-        color = MaterialTheme.theme.colors.primaryUi05,
-        startIndent = 16.dp,
-        modifier = modifier,
+    Image(
+        painter = painter,
+        contentDescription = null,
+        modifier = Modifier
+            .graphicsLayer {
+                compositingStrategy = CompositingStrategy.Offscreen
+                this.alpha = alpha
+                this.rotationZ = rotation
+            }
+            .padding(start = offsetX, top = offsetY)
+            .size(width, height)
+            .drawWithContent {
+                drawContent()
+                drawRect(
+                    brush = Brush.verticalGradient(
+                        colors = listOf(gradientColors.first, gradientColors.second),
+                    ),
+                    blendMode = BlendMode.SrcAtop,
+                )
+            },
     )
 }
 
+private val ThemeType.heartColors
+    get() = when (this) {
+        ThemeType.LIGHT, ThemeType.DARK, ThemeType.EXTRA_DARK, ThemeType.ELECTRIC -> blueHeart
+        ThemeType.CLASSIC_LIGHT, ThemeType.ROSE -> redHeart
+        ThemeType.INDIGO -> indigoHeart
+        ThemeType.DARK_CONTRAST -> grayHeart
+        ThemeType.LIGHT_CONTRAST -> blackHeart
+        ThemeType.RADIOACTIVE -> greenHeart
+    }
+
+private val blackHeart = Color.Black to Color(0xFF6B7273)
+private val blueHeart = Color(0xFF03A9F4) to Color(0xFF50D0F1)
+private val redHeart = Color(0xFFF43769) to Color(0xFFFB5246)
+private val indigoHeart = Color(0xFF5C8BCC) to Color(0xFF95B0E6)
+private val greenHeart = Color(0xFF78D549) to Color(0xFF9BE45E)
+private val grayHeart = Color(0xFFCCD6D9) to Color(0xFFE5F7FF)
```

Comment:
The heart image looks great. This might be more of a question for the designers but I wonder if these should be a common theme icon gradients if they are going to be using them in other places.

Reply:
Possibly. I'll keep it as is for now as it's rather easy to refactor if needed.